src/transformers/models/idefics/modeling_idefics.py

```diff
@@ -261,7 +261,7 @@ def freeze_model(model, module_exceptions=[]):
     }
     module_exceptions_mapped = [mapping[m] for m in module_exceptions]
     for module in model.modules():
-        if module_exceptions and any([isinstance(module, t) for t in module_exceptions_mapped]):
+        if module_exceptions and any(isinstance(module, t) for t in module_exceptions_mapped):
```

Comment:
I think this should not be part of this PR

Reply:
This got added automatically when I ran make repo consistency should i remove this?

Reply:
Yes, please. 🙏
This way we keep only the original bug fixing in this PR.